samples/star/src/main/kotlin/com/slack/circuit/star/petdetail/PetDetail.kt

```diff
@@ -158,28 +132,20 @@ constructor(
     return state
   }
 
+  @com.slack.circuit.codegen.annotations.CircuitInject(PetDetailScreen::class, AppScope::class)
```

Comment:
we don't need the whole package name here?

Reply:
IDE refactoring weirdness, will fix

Reply:
9eb69c5